website/docs/patterns/button-alignment/partials/specifications/alignment.md

```diff
@@ -0,0 +1,25 @@
+## Alignment
+
+Alignment refers to how buttons and actions are aligned relative to the container or page they are used in. When combined with different reading patterns and hierarchy focal points, different alignment methods impact the readability, usability, and intended outcome of a function.
+
+- **Left (start) alignment:** align actions to the left of the container or page.
+- **Right (end) alignment:**  align actions to the right of the container or page.
+- **Center alignment:** center actions in the middle of the container or page.
```

Comment:
Do you have visual examples of when someone _might_ use either of these situations?